src/ImageSharp/Common/Helpers/SimdUtils.HwIntrinsics.cs

```diff
@@ -554,6 +555,34 @@ public static class HwIntrinsics
             return Avx.Add(Avx.Multiply(vm0, vm1), va);
         }
 
+        /// <summary>
+        /// Performs a multiplication and an addition of the <see cref="Vector128{Single}"/>.
+        /// TODO: Fix. The arguments are in a different order to the FMA intrinsic.
+        /// </summary>
+        /// <remarks>ret = (vm0 * vm1) + va</remarks>
+        /// <param name="va">The vector to add to the intermediate result.</param>
+        /// <param name="vm0">The first vector to multiply.</param>
+        /// <param name="vm1">The second vector to multiply.</param>
+        /// <returns>The <see cref="Vector256{T}"/>.</returns>
+        [MethodImpl(InliningOptions.AlwaysInline)]
+        public static Vector128<float> MultiplyAdd(
+            in Vector128<float> va,
+            in Vector128<float> vm0,
+            in Vector128<float> vm1)
+        {
+            if (Fma.IsSupported)
+            {
+                return Fma.MultiplyAdd(vm1, vm0, va);
+            }
+
+            if (AdvSimd.IsSupported)
+            {
+                return AdvSimd.Add(AdvSimd.Multiply(vm0, vm1), va);
+            }
+
+            return Avx.Add(Avx.Multiply(vm0, vm1), va);
```

Comment:
```suggestion
            return Sse.Add(Sse.Multiply(vm0, vm1), va);
```

Reply:
> Why do you want to change this to sse?

These methods are defined at `Sse`.
`Avx : Sse2 : Sse` as hierarchy chain (excerpt) thus here the inherited member is used.
Personally I prefere using the type that defined the members, so I see "requirement is SSE" (and AVX isn't required for this method).

> be on pair with the other methods

Ah, that's what I missed here. Leaving the decision to you or the team.
Consistency is good. Maybe these should be cleaned up (changed to SSE where possible, but not as part of this PR).

Reply:
Ty for the explanation. Yeah I'll wait for @JimBobSquarePants 🙂

Reply:
Yeah, you always want to use the lowest supported syntax. The IDE usually reminds me to do so when I haven't, dunno if that's Roslynator or built in analyzers.